packages/core/upload/admin/src/components/TableList/utils/isSelectable.js

```diff
@@ -0,0 +1,13 @@
+export const isSelectable = (allowedTypes, elementType, fileType, isFolderSelectionAllowed) => {
```

Comment:
lets write some jsdoc types for this function, also would it benefit from some defaults?

Reply:
mmh I wonder what default result we would want, false?
I refactored it in #14998 could I make the jsdoc types there instead? 👀

Reply:
yes 👍🏼